### ISSUE TITLE
Copy url context when copying text to the clipboard

### ISSUE THREAD

**Issue by deathau** [deathau/markdown-clipper#11]
Inspired by good old OneNote, whenever you copy markdown from the clipper, it would be great if it appended a link to the source to the copied text.

This would have to be an option, so you could disable it if you didn't want it.

**Comment by deathau** [comment]
Could be related to #9 where the front-matter (or back-matter?) gets appended to any text you copy.

Again, an option to disable this behaviour if you don't want it though.

**Comment by sambbaron** [comment]
Great feature from OneNote.

> Could be related to #9 where the front-matter (or back-matter?) gets appended to any text you copy.

I would say not quite. I would prefer having front/back-matter for saving documents separate from the reference for clipped/selected content. The front/back-matter includes a lot of metadata, and the clipped reference is usually just the url.

It would be great if the selected text reference could also be editable with tokens like the front/back-matter.

**Comment by deathau** [comment]
Yeah, like a separate "clipboard reference template" or something. This makes sense. Thanks for the suggestion 👍
